python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -39,7 +39,7 @@ class LocalFile(IGangaFile):
                                      })
     _category = 'gangafiles'
     _name = "LocalFile"
-    _exportmethods = ["location", "remove", "accessURL", "processOutputWildcardMatches", "hasMatchedFiles"]
+    _exportmethods = ["location", "remove", "accessURL"]
```

Comment:
I think I agree with this removal but is it safe? Are we sure that people aren't using these methods?